modules/ROOT/pages/manage-proxy.adoc

```diff
@@ -62,11 +51,22 @@ rtfctl apply http-proxy "http://<user>:<pass>@<10.0.0.1>:<8080>" --no-proxy "<1.
 The password must be URL encoded.
 * *<10.0.0.1>*: the IP address or hostname to access the HTTP proxy.
 * *<8080>*: the port on the host where the HTTP proxy is listening for requests.
-* *<1.1.1.1:8888,2.2.2.2.:9999>*: the `NO_PROXY` hosts and ports, delimited by commas.
+* *<1.1.1.1:8888,2.2.2.2:9999>*: the `NO_PROXY` hosts and ports, delimited by commas.
 . To verify the change was successful, run the following command to output the current value of the HTTP proxy: `rtfctl get http-proxy`.
 
 The output should match the value expected.
 
+=== Run the Script on Each Node
+
+. Copy the `set_dockerd_proxy.sh` from the controller node used to install Runtime Fabric, which is located under `/opt/anypoint/runtimefabric/scripts/`, to each controller and worker node under the same location (`/opt/anypoint/runtimefabric/scripts/`). +
+[NOTE]
+If you are not able to locate this script, please open a support ticket for assistance.
+. Update the `/opt/anypoint/runtimefabric/env` file to set the values for `RTF_HTTP_PROXY` and `RTF_NO_PROXY` variables. + 
+The `RTF_HTTP_PROXY` value accepts authentication values in-line. For example: `http://user:Password@1.1.1.1:80` +
+The `RTF_NO_PROXY` value accepts a comma-separated list of hosts that should not go through the proxy. For example: `1.1.1.1,no-proxy.com`
+. Run the script `/opt/anypoint/runtimefabric/scripts/set_dockerd_proxy.sh`.
+. After repeating these steps on each controller and worker node, apply these proxy values to the services running within Runtime Fabric.
```

Comment:
Is this actually part of the Task identified for this list of steps or is this the introduction to the next set of steps?
If the latter, format as plain text, or put under a "Next Steps" title, if you like.